src/cordova/www/css/index.css

```diff
@@ -1,5 +1,62 @@
-﻿body {
-    -webkit-touch-callout: none;
-    -webkit-text-size-adjust: none;
-    -webkit-user-select: none;
-}
\ No newline at end of file
+    body {
```

Comment:
I followed all your instructions mentioned above, but i am not able to see any change in the cordova app folder "src\cordova\www\css" Am I missing something here?